.coderabbit.yaml

```
@@ -25,5 +26,9 @@ reviews:
         base_branches:
             - "develop"
             - "feature/.*"
+        exclude_branches:
+            - "master"
+            - "hotfix/.*"
+            - "release/.*"            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)